composer.json

```diff
@@ -158,7 +158,6 @@
         "sylius/user-bundle": "self.version"
     },
     "conflict": {
-        "api-platform/core": "^2.6",
```

Comment:
If you did some changes specific for api-platform/core in 2.6 version, I would be also for changing this dependency from `"api-platform/core": "^2.5"` to `"api-platform/core": "^2.6",`